### ISSUE TITLE
Convert filenames to capital case for component names

### ISSUE THREAD

**Issue by csandman** [skyran1278/js-jsx-snippets#5]
I, like many people, use kebab-case for all of my filenames, however when using the shortcuts for creating a component, it maintains that case for the component's name:

```js
import React from 'react';

const test-component = () => {
  return (
    <div>
      
    </div>
  );
}

export default test-component;
```

It would be great if it could convert the filename to capitol case when using it to name the component, as dashes aren't valid in variable names, and because react components need to start with a capitol letter to be valid.

**Comment by skyran1278** [comment]
Thanks your feedback, I will address this issue soon.

**Comment by skyran1278** [comment]
fixed by version 11.0.0
